.jenkins/Jenkinsfile

```diff
@@ -201,7 +200,11 @@ def win2016DebugCrossCompile() {
         node('SGXFLC-Windows') {
             cleanWs()
             checkout scm
-            PowerShellWrapper('./scripts/test-build-config.ps1 -build_type Debug -build_enclaves')
+            dir('build') {
+                withEnv(["CC=cl.exe","CXX=cl.exe"]) {
+                    bat "vcvars64.bat x64 && cmake.exe -G Ninja -DCMAKE_BUILD_TYPE=Debug -DBUILD_ENCLAVES=1 .. && ninja.exe && ctest.exe -V -C Debug"
+                }
+            }
        }
```

Comment:
The builds should occur in the build/X64-Debug directory for the Debug Cross Compile with ninja.

Reply:
Hey, I noticed your changes in the script, but is there any reason it needs to be in that specific directory though?

We are cleaning the Jenkins workspace before every build, so no wires'll get crossed, and it seems to build just fine like this.

Reply:
Our CI run should mimic what we are suggesting developers to follow: https://github.com/Microsoft/openenclave/blob/master/docs/GettingStartedDocs/GettingStarted.Windows.md

Yes, we want the debug builds and release builds to be in the directories X64-Debug/X64-Release respectively. For consistency, let us change the directory for the Linux-Elf-Build configurations to be similar. Thanks.

Reply:
Gotcha! Done!